### PR TITLE
[TF] Fix SR-8358: parameter aggregate updater not being GPE'd 

### PR DESCRIPTION
**Pull request by rxwei**
Public functions containing tensor code in the `TensorFlow` module **must be inlinable**.

https://bugs.swift.org/browse/SR-8358

### CODE REVIEW

File: test/TensorFlowRuntime/parameter_update.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// REQUIRES: tensorflow_swift_bindings
+//
+// TensorFlow aggregate parameter update tests.
+
+import TensorFlow
+import StdlibUnittest
+
+var ParameterUpdateTests = TestSuite("ParameterUpdate")
+
+RawOpTests.test("UpdateParameters") {
```
Comment (dan-zheng) [inline test/TensorFlowRuntime/parameter_update.swift]: Rename `RawOpTests` to `ParameterUpdateTests`

File: test/TensorFlowRuntime/parameter_update.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+// REQUIRES: swift_test_mode_optimize
+// REQUIRES: tensorflow_swift_bindings
```
Comment (dan-zheng) [inline test/TensorFlowRuntime/parameter_update.swift]: Remove `REQUIRES: tensorflow_swift_bindings`

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Review by dan-zheng — APPROVED** [review]

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please clean test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

File: test/TensorFlowRuntime/parameter_update.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// REQUIRES: swift_test_mode_optimize
+//
+// TensorFlow aggregate parameter update tests.
+
+import TensorFlow
+import StdlibUnittest
+
+var ParameterUpdateTests = TestSuite("ParameterUpdate")
+
+// TODO: When SR-8360 is fixed, move this to the body of the 'UpdateParameters'
```
Comment (dan-zheng) [inline test/TensorFlowRuntime/parameter_update.swift]: Undo SR-8360 workaround.

Comment (rxwei) [reply test/TensorFlowRuntime/parameter_update.swift]: Alright.

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Pull request merged**